jobs-managing-service/src/main/resources/application.yml

```diff
@@ -29,6 +29,8 @@ spring:
 
 server:
   port: '8081'
+  user: 'admin'
+  password: '$2y$08$yrbEY7O0.WjYnGolNmHJN.ymasX5ETmidei162v87yhygZFoEcLHu'
```

Comment:
I assume, all services and share the same authentication data, right? 
Later when we have  database to store credentials they will all check the authentication against that database. Is there, in the mean time, a easy measure to avoid repeatedly putting user and password into every `application.yml` file`